design/clusters.go

```diff
@@ -78,4 +108,16 @@ var _ = a.Resource("clusters", func() {
 		a.Response(d.InternalServerError, JSONAPIErrors)
 		a.Response(d.Unauthorized, JSONAPIErrors)
 	})
+
+	a.Action("create", func() {
+		a.Security("jwt")
+		a.Routing(
+			a.POST("/"),
+		)
+		a.Payload(createCluster)
+		a.Description("Add a cluster configuration")
+		a.Response(d.Created)
+		a.Response(d.Unauthorized, JSONAPIErrors)
+		a.Response(d.InternalServerError, JSONAPIErrors)
```

Comment:
Add 400 Bad Request. And also tests for that :)

Reply:
good point, indeed!

Reply:
done in 7ba3711